packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -54,6 +57,73 @@ List<String> debugDescribeTransform(Matrix4 transform) {
   return matrix;
 }
 
+bool _extensionsInitialized = false;
+
+void initServiceExtensions() {
```

Comment:
I'm not sure this is the right place for this code.  We should probably put it in its own file.

Also, we shouldn't execute this code in production.  Maybe the whole thing should run only in checked mode?